pkg/handlers/document/bulk.go

```diff
@@ -143,7 +143,7 @@ func BulkWorker(target string, body io.Reader) (*BulkResponse, error) {
 			default:
 			}
 
-			newIndex, _, err := core.GetOrCreateIndex(indexName, "")
+			newIndex, _, err := core.GetOrCreateIndex(indexName, "", 0)
```

Comment:
An explanation of what shard_num 0 will do?

Reply:
if shard_num set 0, it will use `config.Global.Shard.Num` as default shard num.

And the default shard num can set by environment: `ZINC_SHARD_NUM`, default is 5.

Reply:
thanks